File: classes/parte-9-ingenieria-de-software-poliglota/139-pruebas-unitarias-por-lenguaje/implementaciones/c/main.c
#include <stdio.h>

int main(void) {
    long a, b, esperado;
    if (scanf("%ld %ld %ld", &a, &b, &esperado) != 3) return 1;
    printf("test=%s\n", a + b == esperado ? "pasa" : "falla");
    return 0;
}

File: classes/parte-9-ingenieria-de-software-poliglota/139-pruebas-unitarias-por-lenguaje/implementaciones/go/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	line, _ := bufio.NewReader(os.Stdin).ReadString('\n')
	f := strings.Fields(line)
	a, _ := strconv.Atoi(f[0])
	b, _ := strconv.Atoi(f[1])
	esperado, _ := strconv.Atoi(f[2])
	res := "falla"
	if a+b == esperado {
		res = "pasa"
	}
	fmt.Printf("test=%s\n", res)
}
